### PR TITLE
*: update cargo.lock

### LINKED ISSUE #15579: Cargo reference should not point to personal repo.

### PR DESCRIPTION
**Pull request by SpadeA-Tang**
### What is changed and how it works?

Issue Number: Close #15579

What's Changed:

```commit-message
update cargo.lock
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by glorv — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

File: Cargo.lock [hunk 1]
```diff
@@ -3108,7 +3108,7 @@ dependencies = [
 [[package]]
 name = "librocksdb_sys"
 version = "0.1.0"
-source = "git+https://github.com/SpadeA-Tang/rust-rocksdb.git?branch=fix-sealed-chaos#f5121f48a1543c5d576ad7964c617f30f79a3d66"
```
Comment (tonyxuqqi) [inline Cargo.lock:3111]: How it's checked-in at first place?

**Review by tonyxuqqi — APPROVED** [review]

File: Cargo.lock (hunk 1 above)
Comment (SpadeA-Tang) [reply Cargo.lock:3111]: seems ci does not check the consistency between cargo.toml and cargo.lock

**Pull request merged**